### ISSUE TITLE
Rules from certain filters only partially loaded

### ISSUE THREAD

**Issue by BetaMatrixCompositor** [AdguardTeam/AdGuardForSafari#642]
### Issue Details

* **Version of AdGuard for Safari:**
  * 1.10.4 (release, non App-Store)
* **Environment name and version:**
  * Safari 14.1.2
* **Operating system and version:**
  * macOS Big Sur 11.5.1
* **Filters you use in AdGuard:**
  * Base, Dutch, German, Spyware, Social, Annoyances, Search Ads

### Expected Behavior
All rules from filters should be loaded

### Actual Behavior
Only 989 rules from Base filter are loaded, 147 from Spyware filter, rest of filters seem fine. See screenshot below, missing rules are also apparent by Safari loading ads.

</details>

### Screenshot
<img width="1072" alt="Screenshot" src="https://user-images.githubusercontent.com/76661735/129071651-c6ede1ff-33ee-48e9-8572-1c04a2e7617a.png">

### Additional Information
Reinstalling AdGuard for Safari did not help, including deleting support files.

**Comment by tvinzz** [comment]
@BetaMatrixCompositor, can you please provide the logs?

**Comment by BetaMatrixCompositor** [comment]
@tvinzz, of course, see the [log](https://github.com/AdguardTeam/AdGuardForSafari/files/6984259/log.log) here.

**Comment by tvinzz** [comment]
@BetaMatrixCompositor, thank you.
After filters updating does it show the same amount?

**Comment by BetaMatrixCompositor** [comment]
@tvinzz If there are actual updates to the filters, it changes the count. It remains the same without actual updates.

From updating just now:
- Before update: 8 filters, 76845 rules, 17279 advanced rules
- After update: 8 filters, 39394 rules, 4969 advanced rules.

**Comment by BetaMatrixCompositor** [comment]
In addition to the first post, completely removing all traces from the stand-alone version and installing the App Store version unfortunately did not help.

**Comment by windwak3r** [comment]
Please install the latest AG Mini version and check if the problem persists. Closing the issue for now.
